methods/build.js

```diff
@@ -8,48 +9,48 @@ module.exports = function (config, settings, data) {
 
   const buildService = async function(item) {
     let before = data.before;
+
     if (item.config.alwaysBuild) {
       before = null;
     }
 
+    const envVars = {
+      USER: data.user,
+      REPO: data.repo,
+      BRANCH: data.branch || data.tag,
+      TOKEN: settings.githubToken,
+      DOCKERFILE: item.config.dockerfile || 'Dockerfile',
```

Comment:
you aren't passing registry (which we should probably rename to namespace in builder)

Reply:
Was just testing that.